scripts/zones/Misareaux_Coast/npcs/qm2.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------
+-- Area: Misareaux_Coast
+-- NPC:  ??? (Spawn Ziphius)
+-- !pos 76 -16 534 25
+-----------------------------------
+
+require("scripts/globals/npc_util")
+local ID = require("scripts/zones/Misareaux_Coast/IDs")
+
+function onTrade(player,npc,trade)
+    local baited = npc:getLocalVar("[Ziphius]Bait Trap")
+
+    -- Trade Slice of Carp
+    if (npcUtil.tradeHas(trade, 16994) and baited == 0) then
+        npc:setLocalVar("[Ziphius]Bait Trap", 1)
```

Comment:
Do players "share" traps? In that, if Player A baits the trap, if Player B checks later, Player B can get the spawn?

Reply:
It seems that way.

Reply:
Teo :point_left:  _wants a cap showing this_

Reply:
me too!

Reply:
Fortunately we have a free week :)